src/debug/debug_impl.ts

```diff
@@ -860,7 +860,7 @@ class ThreadManager {
 
 	removeStoredIds(ids: number[]) {
 		for (let id of ids) {
-			delete this.storeData[id];
```

Comment:
Yup, looks like. I'm guessing this would have manifested mostly as a memory leak?